msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/entityproviders/ForumsEntityProviderImpl.java

```diff
@@ -501,10 +501,10 @@ public Object displayMessages(EntityView view, Map<String, Object> params) {
 		for(Message fm : (List<Message>) forumManager.getRecentDiscussionForumThreadsByTopicIds(topicIds, numberOfMessages)) {
 			//message has user_Id set in the 'modifiedBy' field, setting it to 'displayId' for display purpose
 			try {
-				String displayName =  userDirectoryService.getUser(fm.getModifiedBy()).getDisplayName();
-				fm.setModifiedBy(displayName);
+				String displayName =  userDirectoryService.getUser(fm.getCreatedBy()).getDisplayName();
+				fm.setCreatedBy(displayName);
 			} catch (UserNotDefinedException e) {
-				LOG.debug(" User not defined for id '" + fm.getModifiedBy() + "'.");
+				LOG.debug(" User not defined for id '" + fm.getCreatedBy() + "'.");
```

Comment:
I think we probably still want the modified by set as well as otherwise if an existing consumer of the API gets this change they won't have the modified by field any more.